charts/dapr/charts/dapr_sidecar_injector/values.yaml

```diff
@@ -1,3 +1,4 @@
+enabled: true
```

Comment:
going back to https://github.com/dapr/dapr/pull/5896, if the default value is going to be true, should we provide a flag to instead `disable` sidecar injector?

Reply:
Would this be a flag on the `dapr` cli? And if so, how would we want to handle the case where there is a conflict? `dapr init -k --disable-sidecar-injector --set dapr_sidecar_injector.enabled=true`

Reply:
sorry I meant have the config option as `dapr_sidecar_injector.disabled` instead of `enabled`.

Reply:
Ah gotcha. I'd much rather we use the [convention `enabled`](https://helm.sh/docs/chart_best_practices/dependencies/#conditions-and-tags) as I see this everywhere in the wild and consider it as a convention.

Reply:
I don't think we should have a Dapr CLI flag `--disable-sidecar-injector`. Seems a niche-enough thing that could be kept as a Helm-chart only option

Reply:
CLI doesn't need a flag.